docs/getting-started/contributors.md

```diff
@@ -10,6 +10,14 @@ repository.
 
 ## Cloning the Repository
 
+Ok. So you're gonna want some code right? Go ahead and fork the repository into
+your own GitHub account and clone that code to your local machine or you can
+grab the one for the origin like so:
+
+```bash
+git clone git@github.com/backstage/backstage --depth 1
+```
+
```

Comment:
Is it worth mentioning how to add an upstream git reference for periodically updating to the latest upstream master?